### PR TITLE
Add install instructions

### PR DESCRIPTION
**Pull request by baderbuddy**

### CODE REVIEW

File: docs/cli.md [hunk 1]
```diff
@@ -3,7 +3,30 @@ meta:
...
-#### CLI Options
+
+## Installation
+
+### From GitHub Releases
+Visit the [releases page](https://github.com/FairwindsOps/polaris/releases) to find the release
+that's right for your environment. For example, on Linux:
+```
+curl -L "https://github.com/FairwindsOps/polaris/releases/download/4.2.0/polaris_linux_amd64.tar.gz" > polaris.tar.gz
+tar -xvf polaris.tar.gz
```
Comment (makoscafee) [inline docs/cli.md:14]: you do not need to `chmod +x polaris`?

**Review by makoscafee — APPROVED** [review]

File: docs/cli.md (hunk 1 above)
Comment (baderbuddy) [reply docs/cli.md:14]: Nope, the tarball has the right permissions inside it.

**Comment by baderbuddy** [comment]
I tried signing the CLA multiple times, bummer it looks broken

**Comment by sudermanjr** [comment]
@baderbuddy  Do the email address for github and the one that you used to make this commit match?

I'm looking at other angles as well.

**Pull request closed without merging**